console-csharp-connect-sample/GraphClient.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Graph;
+using Microsoft.Identity.Client;
+using System.Linq;
+using console_csharp_connect_sample.Helpers;
+
+namespace console_csharp_connect_sample
+{
+	// This class holds the graph client data and 
+	// returns a fully constructed instance of the 
+	// GraphServiceClient with the client data to be 
+	// used when authenticating requests to the Graph API
+	public class GraphClient
```

Comment:
Rename this class to GraphClientFactory.  Make it a static class.  Move the parameters in the ctor to the method GetGraphServiceClient and make that method static.

Reply:
Done. Thanks @darrelmiller